lib/config/migration.ts

```diff
@@ -251,6 +251,13 @@ export function migrateConfig(
               'masterIssue',
               'dependencyDashboard'
             );
+          } else if (
+            is.string(val[i]) &&
+            (val[i] === 'default:unpublishSafe' ||
```

Comment:
I think it would most commonly be `:unpublishSafe` and not `default:unpublishSafe`. We need to migrate both patterns